tests/integration/it_owner_privileges_test.py

```diff
@@ -142,3 +142,101 @@ def test_can_not_manage_locations_from_different_company(clean_app, db_session):
     assert not has_privilege(
         method=Method.DELETE, resource="location", id=location.id
     )
+
+
+def test_can_manage_users_from_same_company(clean_app, db_session):
```

Comment:
@paulodamaso test's name does not reflect idea what is happening below.
here is not only `manage users from same company`. Pls change it or add comments to clarify what test is about.